JavaScript/Topics/3.Web_Browser/HTML/2.index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Internal JS</title>
+</head>
+<body>
+    <h1 id = "main"></h1>
+
+    <script>
+        "use strict";
+        let x = document.getElementById("main");
+        x.innerHTML = "Internal JavaScript";
```

Comment:
```suggestion
        x.textContent = "Internal JavaScript";
```
innerHTML is also correct but why to use it if we are not setting an HTML element, for text we should textContent, always use right tool for the a specific work